src/service/viewport-impl.js

```diff
@@ -1771,6 +1771,12 @@ function getViewportType(win, viewer) {
   if (!isIframed(win) && (getMode(win).localDev || getMode(win).development)) {
     return ViewportType.NATURAL_IOS_EMBED;
   }
+
+  // Enable iOS Embedded mode for iframed tests (e.g. integration tests).
```

Comment:
@dvoytenko When running integration tests in iOS 10, I was getting the Natural instead of Embed iOS viewport. That was causing certain parts of the test (around scrolling and viewability) to get the wrong values. I think this change makes sense, please confirm.